.env.example

```
@@ -9,11 +9,24 @@
 # ============================================================================
 # Application URL (used for referral links and redirects)
 NEXT_PUBLIC_APP_URL=https://babylon.market
+# Waitlist URL (canonical landing for referrals and "no access" redirects when app is token-gated)
+# In production: https://babylon.market
+# In staging: https://staging.babylon.market
+NEXT_PUBLIC_WAITLIST_URL=https://babylon.market
 # Blog URL (used for the blog link on the landing page)
 NEXT_PUBLIC_BLOG_URL=https://blog.babylon.market
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_BLOG_URL key should go before the NEXT_PUBLIC_WAITLIST_URL key

(UnorderedKey)